src/ui/dialogs/contactsearch.ts

```diff
@@ -0,0 +1,253 @@
+import Dialog from '../Dialog'
+import Contact from '../../Contact'
+import Form from '../../connection/Form'
+import Log from '../../util/Log'
+import { IConnection } from '@connection/Connection.interface';
+import Translation from '@util/Translation';
+import Client from '../../Client'
+import Account from '../../Account'
+import JID from '../../JID'
+
+let dialog: Dialog;
+
+export const CANCELED = 'canceled';
+
+let contactSearchTemplate = require('../../../template/contactsearch.hbs');
+
+export default function(username?: string) {
+
+    let content = contactSearchTemplate({
+      accounts: Client.getAccountManager().getAccounts().map(account => ({
+         uid: account.getUid(),
+         jid: account.getJID().bare,
+      }))
+    });
+
+    dialog = new Dialog(content);
+    dialog.open();
+
+    let accountId = <string> $('#jsxc-account').val();
```

Comment:
I guess this is the same as `account.getUid()`.

Reply:
let accountId = <string> $('#jsxc-account').val();
let account = Client.getAccountManager().getAccount(accountId);

but need accountId to get account

Reply:
I guess it's the first account by default. Am I right?

Reply:
we could also use: 
let account = Client.getAccountManager().getAccount()[0];

Reply:
changed...